src/main/java/pages/AbstractPage.java

```diff
@@ -0,0 +1,10 @@
+package pages;
```

Comment:
looks like AbstractPage is an extra level of abstraction and you may you use TestedPage instead